src/Command/DynamicMetadata/ImageDynamicMetadataDeleteSubjectAreaCommand.php

```diff
@@ -14,7 +14,7 @@ class ImageDynamicMetadataDeleteSubjectAreaCommand extends BaseRokkaCliCommand
     protected function configure()
     {
         $this
-            ->setName('image:delete-subjectarea')
+            ->setName($this->namePrefix.'image:delete-subjectarea')
```

Comment:
Can't we override the `BaseRokkaCliCommand::setName()` to have the prefix appended by default?
Do you see any issue with that?